packages/blade/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -191,23 +185,32 @@ export default {
       },
       mapping: tabItemTrailing,
       options: Object.keys(tabItemTrailing),
-      defaultValue: 'Badge',
       description: 'Trailing element of the tab item. Can be used to render an badge/counter.',
     },
     tabItemHref: {
       table: { category: propsCategory.TAB_ITEM },
       control: {
         type: 'text',
       },
-      defaultValue: '',
     },
     isLazy: {
       table: { category: propsCategory.TABS },
-      defaultValue: false,
     },
   },
+  args: {
+    isFullWidthTabItem: false,
+    orientation: 'horizontal',
+    size: 'medium',
+    variant: 'bordered',
+    tabItemChildren: 'Tab Item',
+    tabItemIsDisabled: false,
+    tabItemLeading: ('ClipboardIcon' as unknown) as IconComponent,
+    tabItemTrailing: 'Badge',
+    tabItemHref: '',
+    isLazy: false,
+  },
   // eslint-disable-next-line babel/new-cap
-  decorators: [StoryRouter(undefined, { initialEntries: ['/accounts/subscriptions'] })],
+  decorators: [StoryRouter(undefined, { initialEntries: ['/accounts/subscriptions'] })] as unknown,
```

Comment:
@anurag-rzp can you check if this is working as intended? Had to make it unknown since this was causing a type issue